prepare.py

```diff
@@ -12,10 +13,36 @@
 import env
 
 
+def _duplicated_image_message_detected(image_name: str, message: str) -> bool:
+    return f"More than one Image exists with the name '{image_name}'" in message
+
+
+def _latest_updated_image_by_name(
+    connection: openstack.connection.Connection, image_name: str
+) -> [openstack.compute.v2.image.Image]:
```

Comment:
Yes, it's not a list but just the most recent image, it was an oversight :/ ... I fixed it. 

`mypy prepare.py` run didn't detect this mistake though - I was thinking on adding a linter for type hints but this case defeats the purpose, WDYT we can do to prevent this ?